index.js

```diff
@@ -15,3 +15,128 @@ Array.from(closes).forEach((close) => {
+    popuplanguage: ['html', 'css', 'javaScript', 'github', 'ruby', 'bootstrap'],
+    popDescription: 'Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum?',
+    image: 'images/Snapshoot-Portfolio-desktop.png',
+  },
+  {
+    project_id: 'project2',
+    name: 'Multi-Post Stories',
+    experiance: ['FACEBOOK', 'Backend Dev', '2015'],
+    description:
+      'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    languages: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    popuplanguage: ['html', 'css', 'javaScript', 'github', 'ruby', 'bootstrap'],
+    popDescription:
+      'Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum?',
+    image: 'images/Snapshoot-Portfolio-desktop-2.png',
+  },
+  {
+    project_id: 'project3',
+    name: 'Facebook 360',
+    experiance: ['Facebook', 'Backend Dev', '2015'],
+    description:
+      'Exploring the future of media in Facebooks first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    languages: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    popuplanguage: ['html', 'css', 'javaScript', 'github', 'ruby', 'bootstrap'],
+    popDescription:
+      'Dolor ipsam molestiae assumenda nisi natus voluptatum Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum?',
+    image: 'images/Snapshoot-Portfolio-desktop3.jpg',
+  },
+  {
+    project_id: 'project4',
+    name: 'Uber Navigation',
+    experiance: ['Uber', 'Backend Dev', '2015'],
+    description:
+      'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    languages: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    popuplanguage: ['html', 'css', 'javaScript', 'github', 'ruby', 'bootstrap'],
+    popDescription:
+      'Dolor ipsam molestiae assumenda nisi natus voluptatum Dolor ipsam molestiae assumenda nisi natus voluptatum Dolor ipsam molestiae assumenda nisi natus voluptatum Dolor ipsam molestiae assumenda nisi natus voluptatum',
+    image: 'images/Snapshoot-Porfolio-desktop4.png',
+  },
+];
+
+const openModalButtons = document.querySelectorAll('[data-modal-target]');
+const closeModalButtons = document.querySelectorAll('[data-close-button]');
+const overlay = document.getElementById('overlay');
+
+function openModal(modal) {
+  if (modal == null) return;
+
+  const modals = document.querySelectorAll('.modal');
+
+  for (let i = 0; i <= projects.length; i += 1) {
```

Comment:
- [ ] Kindly be noticed that i should be less than projects.length `i < projects.length` not less than or equal `i <= projects.length`.